index.js

```diff
@@ -54,13 +54,11 @@ function underPressure (fastify, opts, next) {
     var mem = process.memoryUsage()
     heapUsed = mem.heapUsed
     rssBytes = mem.rss
-    var toCheck = now()
-    eventLoopDelay = toCheck - lastCheck - sampleInterval
-    lastCheck = toCheck
   }
 
   function onRequest (req, res, next) {
-    if (checkMaxEventLoopDelay && eventLoopDelay > maxEventLoopDelay) {
+    if (checkMaxEventLoopDelay && loopSampler.delay > maxEventLoopDelay) {
+      loopSampler = loopDelay()
```

Comment:
I am not sure about this

Reply:
No you should not be doing this. Why are you doing it?

Reply:
>No you should not be doing this. Why are you doing it?

In docs it says that it is recording cumulative delay since the creation of the sampler. So if I am thinking correctly it will always increase and will eventually pass the limit, then stay above the limit. Thats why I am a bit confused.